File: 0989-add-to-array-form-of-integer/0989-add-to-array-form-of-integer.java
import java.math.BigInteger;
class Solution {
    public List<Integer> addToArrayForm(int[] num, int k) {
        StringBuilder s = new StringBuilder();
 
        for (int i : num) s.append(i);
        
 
       return  String.valueOf((new BigInteger(s.toString())).add(new BigInteger(String.valueOf(k)))).chars().mapToObj(c -> c - '0').collect(Collectors.toList());
 
    }
}

File: 0989-add-to-array-form-of-integer/0989-add-to-array-form-of-integer.py
import sys
sys.set_int_max_str_digits(0)
class Solution:
    def addToArrayForm(self, num: List[int], k: int) -> List[int]:
        s="".join(str(x) for x in num)
        an=str(int(s)+k)
        return [int(x) for x in an]
        